src/Common/tests/System/PlatformDetection.cs

```diff
@@ -158,9 +158,12 @@ public static bool IsWinRT
         public static bool IsWindowsSubsystemForLinux => m_isWindowsSubsystemForLinux.Value;
         public static bool IsNotWindowsSubsystemForLinux => !IsWindowsSubsystemForLinux;
 
-        public static bool IsNotFedoraOrRedHatOrCentos => !IsDistroAndVersion("fedora") && !IsDistroAndVersion("rhel") && !IsDistroAndVersion("centos");
+        public static bool IsNotFedoraOrRedHatFamily => !IsDistroAndVersion("fedora") && !IsRedHatFamily;
 
         public static bool IsFedora => IsDistroAndVersion("fedora");
+        public static bool IsRedHatFamily => IsRedHatFamilyAndVersion();
+        public static bool IsRedHatFamily7 => IsRedHatFamilyAndVersion("7");
```

Comment:
For CentOS and RHEL 7, there is a difference in the version reported via /etc/os-release. CentOS reports just 7, but RHEL reports 7.x where x is the minor version. So this needs to be taken into account, unless I am missing something, there is no version normalization code anywhere.
That is different for version 6 where there is no /etc/os-release and the /etc/redhat-release contains version 6.x, that means including the minor version for both RHEL and CentOS.

Reply:
ok, i've added additional checks in IsRedHatFamilyAndVersion and RedHatVersionEqual